### DISCUSSION TITLE
posts/bahasa-pemrograman-tidak-penting

### DISCUSSION THREAD

**Discussion by giscus** [teknologi-umum/blog#144] [General]
# posts/bahasa-pemrograman-tidak-penting

Terlalu banyak bahasa pemrograman hanya membuat kita pusing.

https://teknologiumum.com/posts/bahasa-pemrograman-tidak-penting

**Comment by woicip** [comment]
Akhirnya ada yang speak up mengenai ini, untuk yang sudah berpengalaman pasti punya pemikiran seperti ini. Bahasa pemrograman bukanlah hal yang penting karena pada dasarnya mereka adalah alat untuk membantu kita membuat suatu hal yang kita butuhkan/inginkan.
